lib/genesis_collector/disks.rb

```diff
@@ -12,8 +12,12 @@ def collect_disks
         d[:product] = info.match(/^(?:Device Model|Product):\s+(.*)$/)[1]
         d[:serial_number] = info.match(/^Serial (?:n|N)umber:\s+(.*)$/)[1] rescue nil
         d[:size] = info.match(/^User Capacity:\s+(.*)$/)[1].split('bytes')[0].strip.gsub(',', '')
-        d[:slot] = get_scsi_slot(d[:dev]) if d[:dev] =~ /^\/dev\/sd/
-        d[:uuid] = disk_uuid(d[:dev]) if d[:dev] =~ /^\/dev\/sd/
+
+        if d[:dev].start_with?("/dev/sd")
+          d[:slot] = get_scsi_slot(d[:dev])
+          d[:uuid] = disk_uuid(d[:dev])
+          d[:status] = "unhealthy" unless disk_healthy(d[:dev])
```

Comment:
The problem is there is no "healthy" status, there's "allocated" and "spare" on our app and I don't think it will handle other stats.